src/docs/v10-vs-v11-retrospective.md

```diff
@@ -0,0 +1,643 @@
+renaming or other refactoring. For example, the fairly vague root class `OmicsProcessing` is now
+`NamedThing.PlannedProcess.DataGeneration`, which now has the following subclasses:
+
+- `MassSpectrometry`
+- `NucleotideSequencing`
+
+The legacy `BiosampleProcessing` class (including `Pooling`, `LibraryPreparation`) and some classes that were previously
+direct subclasses of `PlannedProcess` (`Extraction`, `SubSamplingProcess`, `MixingProcess`, `FiltrationProcess`,
+`ChromatographicSeparationProcess`) have been re-rooted into a new `MaterialProcessing` class.
+
+`DissolvingProcess` and `ChemicalConversionProcess` are new subclasses of `MaterialProcessing`.
+
+### v11.0.1
+
+![MaterialProcessing.png](../images/MaterialProcessing.png)
+
+`StorageProcess` has been added as a new  `PlannedProcess` but the schema does not classify it as a
+`MaterialProcessing` because it does not intrinsically create new/modified samples.
+
+Terms that implied quantification like `MetaboliteQuantification` have been renamed to reemphasize the more fundamental
+activity of identifying, thus `MetaboliteIdentification`
+
+A `CalibrationInformation` class, a `Configuration` class and two `Configuration` subclasses (illustrated below) have
+been added in support of chromatographic separations and mass spectrometry analyses. These are subclasses of the
+new `InformationObject`, which has also become the parent of `DataObject`, which used to be a direct subclass
+of `NamedThing`.
+
+### v11.0.1
+
+![InformationObject.png](../images/InformationObject.png)
+
+### v11.0.1
+
+![Configuration.png](../images/Configuration.png)
+
+Solution-centric modelling for laboratory processes was also replaced with substance-centric modelling
+
+### Removed:
+
+- `Solution`
+- `SolutionComponent`
+
+### Added:
+
+- `PortionOfSubstance`
+- `MobilePhaseSegment`
+
+### v11.0.1
+
+![PortionOfSubstance.png](../images/PortionOfSubstance.png)
+
+### v11.0.1
+
+![MobilePhaseSegment.png](../images/MobilePhaseSegment.png)
+
+The Berkeley schema retains the `AttributeValue` hierarchy but eliminates classes like `IntegerValue`, and `UrlValue`
+whose only advantages over a values of integer or string types were never-used provenance slots.
+
+### v10.9.0
+
+![AttributeValue10.png](../images/AttributeValue10.png)
+
+### v11.0.1
+
+![AttributeValue11.png](../images/AttributeValue11.png)
+
+Classes related to metabolic reactions that a cell could carry out, based on some functionality in its genome, have been
+removed, due to potential confusion with reactions that an experimenter might carry out in order to amke a sample
+suitable for analysis.
+
+### Removed:
+
+- `Reaction`
+- `ReactionParticipant`
+
+### Added
+
+- `ChemicalConversionProcess`
+
+A very general `Instrument` class was added as a subclass of `MaterialEntity` , but without any subclasses. Knowledge
+about instruments is captured directly in NMDC's MongoDB. The instances are normalized by populating their `vendor` slot
```

Comment:
```suggestion
about instruments is captured directly in NMDC's MongoDB database. The instances are normalized by populating their `vendor` slot
```

Reply:
Always thought the DB in MongoDB meant database... if it doesn't then 🤯
If it does, then @eecavanna  suggestion is redundant

Reply:
I, too, assume the "DB" in MongoDB does stand for database; but there are "MongoDB" things that are not databases. There are MongoDB clients (e.g. Mongoose, Studio 3T, mongosh) and the MongoDB server software, for example. To disambiguate from those things, I recommend the article say "the MongoDB database" (or, colloquially, "the Mongo database"), but not "the MongoDB". The latter may lead people to wonder "the MongoDB _what_?"

Reply:
either is fine with me

Reply:
thanks @eecavanna ! I agree with this change then